### PR TITLE
use Subject Alternative Names from cert as immediate callerid groups

### PR DESCRIPTION
**Pull request by acharis**
specifically only the DNSNames

(i was unable to get usable test coverage information for server.go, hints appreciated)

### CODE REVIEW

**Comment by bbeaudreault** [comment]
Looks like travis may have found some tests for you. Other than that the approach LGTM

File: go/vt/vtgate/grpcvtgateservice/no_test.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+import (
+	"testing"
+)
+
+// This package is mostly tested from VTGate. This place-holder test
+// is to document this fact. In order to make sure all code paths are
+// covered, you need to run the coverage tool from the vtgate directory
+// using the following command:
+//	go test -coverprofile=c.out -coverpkg ./engine,.
```
Comment (sougou) [inline go/vt/vtgate/grpcvtgateservice/no_test.go]: `go test -coverprofile=c.out -coverpkg ../grpcvtgateservice,.` (not ./engine)

File: go/vt/vtgate/grpcvtgateservice/no_test.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+limitations under the License.
+*/
+
+package grpcvtgateservice
+
+import (
+	"testing"
+)
+
+// This package is mostly tested from VTGate. This place-holder test
```
Comment (sougou) [inline go/vt/vtgate/grpcvtgateservice/no_test.go]: from ../grpcvtgatesconn

File: go/vt/vtgate/grpcvtgateservice/no_test.go [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+// is to document this fact. In order to make sure all code paths are
+// covered, you need to run the coverage tool from the vtgate directory
```
Comment (sougou) [inline go/vt/vtgate/grpcvtgateservice/no_test.go]: ../grpcvtgateconn

**Comment by sougou** [comment]
@alainjobart  for the cert name change.

**Comment by alainjobart** [comment]
LGTM

**Comment by alainjobart** [comment]
Can be merged as soon as travis passes.

**Comment by bbeaudreault** [comment]
all tests passing now.  thanks @alainjobart and @sougou

**Pull request merged**